### ISSUE TITLE
RNFS.copyFile() nerver resolves

### ISSUE THREAD

**Issue by GabrielFreitasYamamoto** [itinance/react-native-fs#734]
I have to copy a file picked with react-native-document-picker to the temporary path of my app and then read this file, but when it reaches the `await ReactNativeFS.copyFile(realPath, tempPath);` line, it never resolves. Here's the code

```
searchAndReadFiles = async () => {
    try { 
        const fileSelected = await DocumentPicker.pick ({
             type: DocumentPicker.types.plainText,
        }); 
        const decodedURI = decodeURIComponent(fileSelected.uri);
	const split = decodedURI.split('/');
	const name = split.pop();
	const inbox = split.pop();
	const realPath = `${ReactNativeFS.TemporaryDirectoryPath}/${name}`;
	const tempPath = `${ReactNativeFS.ExternalStorageDirectoryPath}/${fileSelected.name}`;
	await ReactNativeFS.copyFile(realPath, tempPath);
	const fileRead = await ReactNativeFS.readFile(tempPath);
    } catch (err) {	
	console.warn(err);
    }
}
```

**Comment by quarryman** [comment]
Have the same issue here.
ReactNativeFS.copyFile(path, pathTo);
neither 'then' nor 'catch' are invoked

**Comment by GabrielFreitasYamamoto** [comment]
Well, I resolved this by doing what I explain here: https://stackoverflow.com/questions/57529614/react-native-fs-copyfile-never-resolves/57597514#57597514
